### PR TITLE
[Calling][Feature] Call screen header custom button

### PR DESCRIPTION
**Pull request by pavelprystinka**
## Purpose

* Call screen header custom button

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull Request Checklist

- [ ] Public API changes
- [ ] Verified for cross-platform
- [ ] Synced with iOS, Web team
- [ ] Internal review completed
- [ ] UI Changes
  - [ ] Screen captures included
  - [ ] Tested for Light/Dark mode
  - [ ] Tested for screen rotation
  - [ ] Tested on Tablet and small screen device (5")
  - [ ] Include localization changes
- [ ] Tests
  - [ ] Memory leak analysis performed
  - [ ] Tested on API 21, 26 and latest 
  - [ ] Tested for background mode
  - [ ] Unit Tests Included
  - [ ] UI Automated Tests Included

## How to Test

* Open...
* Click on...

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by ShaunaSong — APPROVED** [review]

**Review by ShaunaSong — APPROVED** [review]

File: azure-communication-ui/build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     ext {
-        call_library_version_name = '1.11.0'
+        call_library_version_name = '1.13.0-beta.1'
```
Comment (ahammer) [inline azure-communication-ui/build.gradle:8]: We should really be using the beta entry for SDK, and using flags to gate Beta API if necessary.

Comment (pavelprystinka) [reply azure-communication-ui/build.gradle:8]: This is UI lib version, not SDK

**Review by iaulakh — APPROVED** [review]

**Review by ShaunaSong — APPROVED** [review]

File: azure-communication-ui/build.gradle (hunk 1 above)
Comment (ahammer) [inline azure-communication-ui/build.gradle:8]: do we want to bump this in develop, or against a beta release branch?

File: azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeExtensions.kt [hunk 2]
```diff
@@ -43,6 +43,10 @@ internal fun CallCompositeCustomButtonViewData.setVisibleChangedEventHandler(han
     this.visibleChangedEventHandler = handler
 }
 
+internal fun CallCompositeCustomButtonViewData.setDrawableIdChangedEventHandler(handler: CallCompositeEventHandler<Int>) {
```
Comment (ahammer) [inline azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeExtensions.kt:46]: missing gating?

Comment (pavelprystinka) [reply azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeExtensions.kt:46]: Yes, it was missed

File: azure-communication-ui/build.gradle (hunk 1 above)
Comment (pavelprystinka) [reply azure-communication-ui/build.gradle:8]: It will be updated in release branch to correct one

**Review by Mohtasim — APPROVED** [review]

**Pull request merged**